File: C/Code Snippetts/primeornot.c
//TO CHECK WHETHER A NUMBER IS PRIME OR NOT
#include<stdio.h>
int main(){
    int a,p=0;
    printf("ENTER A NUMBER: ");
    scanf("%d",&a);
    for(int i=2;i<a;i++){
      if(a%i==0){
        p=1;
      }
    }
    if(p==0){
        printf("GIVEN NUMBER IS PRIME");
    }
    else{
        printf("GIVEN NUMBER IS COMPOSITE");
    }
}

File: Python/Code Snippets/prime or not.py
#TO CHECK WHETHER A NUMBER IS PRIME OR NOT
#AUTHOR: ASWIN P
#DATE: 12/07/2022

#FUNCTION
def prime(n):
    for i in range(2,n//2+1):
        if n%i==0:
            return False
    else:
        return True

#PROGRAM
a=int(input("ENTER THE NUMBER: "))
x=prime(a)
if x==True:
    print(a,"is a PRIME number!!!")
else:
    print(a,"is NOT a PRIME number!!!")